FBControlCore/Diagnostics/FBBatchLogSearch.m

```diff
@@ -197,8 +197,8 @@ - (id)jsonSerializableRepresentation
   for (NSArray *key in self.mapping) {
     mappingDictionary[key] = [self.mapping[key] valueForKey:@"jsonSerializableRepresentation"];
   }
-  BOOL lines = self.options & FBBatchLogSearchOptionsFullLines;
-  BOOL first = self.options & FBBatchLogSearchOptionsFirstMatch;
+  BOOL lines = (self.options & FBBatchLogSearchOptionsFullLines) ? YES : NO;
```

Comment:
Why do we need this change here? Is it an automatically generated code?

Reply:
It was needed to resolve warnings about deprecations.